tests/system/small/test_dataframe.py

```diff
@@ -3385,6 +3385,8 @@ def test_df_to_orc(scalars_df_index, scalars_pandas_df_index):
     ],
 )
 def test_df_value_counts(scalars_dfs, subset, normalize, ascending, dropna):
+    if pd.__version__.startswith("1."):
+        pytest.skip("pandas 1.x produces different column labels.")
```

Comment:
insert update expected value and assert?

Reply:
I don't think its worth the tests complexity - we're validating against 2.x behavior and I think that is enough.